Xamarin.Forms.Core/DeviceInfo.cs

```diff
@@ -43,9 +63,13 @@ protected virtual void Dispose(bool disposing)
 
 		protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
 		{
-			PropertyChangedEventHandler handler = PropertyChanged;
-			if (handler != null)
-				handler(this, new PropertyChangedEventArgs(propertyName));
+			PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
 		}
 	}
+
+	public enum ScreenOrientation
```

Comment:
This exposes only two which should be the lowest denominators. I need to add `Other` so the enum defaults to it. If a future update on any platform adds new orientations and `Portrait` and/or `Landscape` become invalid, then an internal XF update can default to `Other`.